neo/Plugins/Plugin.cs

```diff
@@ -97,7 +97,8 @@ internal static void LoadPlugins(NeoSystem system)
             if (!Directory.Exists(pluginsPath)) return;
             foreach (string filename in Directory.EnumerateFiles(pluginsPath, "*.dll", SearchOption.TopDirectoryOnly))
             {
-                Assembly assembly = Assembly.LoadFile(filename);
+                var file = File.ReadAllBytes(filename);
+                Assembly assembly = Assembly.Load(file);
```

Comment:
Very interesting solution... since sizes are very low, it shouldn't impact memory.